### PR TITLE
Draft proposal for Use Cases

### PR DESCRIPTION
**Pull request by daniel-utilityapi**
> By making a contribution to this repository, I agree to the terms indicated on [this repository](../1_Contributor-License-Agreement.md)

This is an update to the Use Cases [page](https://carbon-data-specification.github.io/Customer-Data/use-cases) on the [website](https://carbon-data-specification.github.io/Customer-Data/), where we disclose which overall use cases we want to address through the specifications this working group writes.

Creating this draft is part of issue #5. We will discuss this pull request as part of the 2022-03-03 working group meeting, and participants not able to attend are encouraged to make comments in this pull request or inline in the pull request's "Files changed" tab.

A [pretty, rendered version](https://github.com/daniel-utilityapi/Customer-Data/blob/maintainer_branch/docs/use-cases.md) of the proposed Use Cases is also available on my maintainer's branch.

Thanks everyone!

--------
## Deadline
Deadline for comments: Thursday, March 17, 2022 (2022-03-17)

## Status
* comments from @halliecramer1 (Google)
* comments from @ssuffian (WattCarbon)
* comments from @steven-ohmconnect (OhmConnect)
* comments from @op-ravi (Microsoft)

### CODE REVIEW

File: docs/use-cases.md [hunk 1]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+and rate plans for the customer for the past 12 months). This data is
+typcially considered private information, so specifications to address this
+use case must consider privacy, consent, and security requirements as part
+of the scope of work.
+
+* **Streamlined User Experience:**
+Many organizations do not have dedicated staff or sufficient resources
+to spend large amounts of time and effort gathering utility data needed
+for carbon accounting calculations. This effectively creates a
+"barrier to entry" that limits carbon accounting to only the most well-resourced
```
Comment (halliecramer1) [inline docs/use-cases.md:46]: Not sure if it's worth mentioning that the barrier to entry is even higher for more granular data, which is needed for more impactful corporate decarbonization goals, or if that is implied in the examples.

File: docs/use-cases.md [hunk 2]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+* Consultants and advisors helping organizations that have climate and decarbonization targets
+
+
+## Use Case 2: Decarbonization Projects <a id="use-case-decarbonization-projects" href="#use-case-decarbonization-projects" class="permalink">🔗</a>
+
+Often, organizations with decarbonization targets (see [Use Case 1: Carbon Accounting](#use-case-carbon-accounting) above),
+will evaluate projects that may positively impact their carbon emissions. These projects
+can range from physical locally (e.g. installing solar or EV charging) and remote (e.g.
+building renewable generation projects via wholesale power purchase agreements), to virtual
+direct (e.g. avoiding marginal emissions by managing building usage dynamically) or financial
```
Comment (halliecramer1) [inline docs/use-cases.md:78]: avoid use of marginal term here - just say "avoided emissions"?

File: docs/use-cases.md [hunk 3]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+Because distributed grid flexibility and load management technologies are active
+participants in managing load on the electrical grid, they often are registered
+as part of a utility or central operator program (e.g. demand response programs).
+Thus, the specifications to address this use case must consider how to include
+program eligibility and participation details as part of the standardized customer
+dataset.
+
+#### Examples of users in this use case:
+
+* Apps and devices that participate in demand response programs (
```
Comment (halliecramer1) [inline docs/use-cases.md:150]: Would consumer DR, such as Google's carbon-aware computing, fall into "grid flexibility" or "decarbonization projects"? I think the answer is both, since there's some overlap, but do we want there to be more of a distinction?

File: docs/use-cases.md [hunk 4]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+
+## Use Case 4: Building Benchmarking <a id="use-case-benchmarking" href="#use-case-benchmarking" class="permalink">🔗</a>
+
+Municipalities and governments are increasingly requiring that building owners submit
+whole-building energy usage information in order to do carbon accounting and measure
+building energy efficiency for their jurisdiction.
+
+#### Needs:
+
+* **Standardized Whole-Building Data Formats:**
```
Comment (halliecramer1) [inline docs/use-cases.md:167]: Are there specifications needed for the other way around? i.e. for multi-tenant buildings, having disaggregation standards?

File: docs/use-cases.md [hunk 5]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+
+* Property owners wanting to benchmark their buildings achieve green and efficiency ratings (
+[Energy Star](https://www.energystar.gov/buildings),
+[LEED](https://www.usgbc.org/leed),
+etc.)
+
+
+## Use cases NOT in scope for this working group <a id="not-in-scope" href="#not-in-scope" class="permalink">🔗</a>
+
+* Live grid operations and controls (actual demand response signaling for equipment, solar inverter control and generation curtailment, EV charging shut-off signals, etc.)
```
Comment (halliecramer1) [inline docs/use-cases.md:216]: For clarity, what additional requirements would be needed for this use case as compared to the current grid flexibility use case, given the "real-time" requirement specified?

File: docs/use-cases.md (hunk 4 above)
Comment (ssuffian) [reply docs/use-cases.md:167]: Those sorts of specs would be super helpful for allocating carbon impacts in multi-tenant buildings, but I'd be a little worried that it may be a bit out of scope initially. Maybe we can include some general guidelines on how someone might go about performing disaggregation without formalizing it as a standard?

**Review by ssuffian — APPROVED** [review]

File: docs/use-cases.md [hunk 6]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+
+#### Needs:
+
+* **Use Case 1 Needs:**
+Like carbon accounting, the calculations for assessing decarbonization
+projects typcially require the specific utility data for a specific entity.
+So, all of the needs for carbon accounting often are needed for assessing
+decarbonization projects, both for the initial feasibility analysis and
+ongoing project measurement and verification ([M&V](https://en.wikipedia.org/wiki/Measurement_and_Verification)).
+
```
Comment (halliecramer1) [inline docs/use-cases.md:89]: I think we should add a time-based energy attribute certificates or granular certificates reference to this use case, e.g. **Time-based energy attribute certificates (TEACs):**  TEACs depend on streamlined access to granular generation data from clean energy projects (via wholesale power purchase agreements). To address this use case, the specifications must consider how to enable and streamline access to the (revenue grade) metered generation data from individual asset owners.

File: docs/use-cases.md [hunk 7]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+#### Needs:
+
+* **Use Case 1 Needs:**
+Like carbon accounting, the calculations for assessing the eligibility of
+deploying or using a distributed grid flexibility or load management technology
+typically requires the specific utility data for a specific entity.
+So, all of the needs for carbon accounting often are needed for assessing
+distributed grid flexibility technologies, both for the initial feasibility
+analysis and ongoing measurement and verification ([M&V](https://en.wikipedia.org/wiki/Measurement_and_Verification)).
+
```
Comment (steven-ohmconnect) [inline docs/use-cases.md:139]: I would suggest adding here (or above as part of the Use Case 1 needs) some mention of additional data elements, specifically customer data elements: including service account numbers, service address, subscription keys, etc... necessary for enrollment in energy markets.

File: docs/use-cases.md [hunk 8]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+* **Program Eligibility and Participation Information:**
+Because distributed grid flexibility and load management technologies are active
+participants in managing load on the electrical grid, they often are registered
+as part of a utility or central operator program (e.g. demand response programs).
+Thus, the specifications to address this use case must consider how to include
+program eligibility and participation details as part of the standardized customer
+dataset.
```
Comment (steven-ohmconnect) [inline docs/use-cases.md:146]: Do we define this "standardized customer dataset" anywhere?

File: docs/use-cases.md [hunk 9]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+is similar to Use Case 1: Carbon Accounting, only usage can be aggregated over
+multiple usage points and be provided as a single combined reading. Thus,
+the specifications to address benchmarkings must consider how to appropriately
+define how data is being aggregated, and where necessary provide meta data
+on the underlying individual components that were used to create the whole-building
+aggregate usage values.
+
+* **Streamlined Data Request and Access Procedures:**
+While individual customer data access will be frequently achieved by the
+user requesting the data directly sending a data request to the customer,
```
Comment (steven-ohmconnect) [inline docs/use-cases.md:178]: This sentence is a little hard to follow. It's unclear the difference between a user and a customer.

**Review by steven-ohmconnect — APPROVED** [review]

File: docs/use-cases.md [hunk 10]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+for carbon accounting calculations. This effectively creates a
+"barrier to entry" that limits carbon accounting to only the most well-resourced
+organizations and prevents widespread adoption of decarbonization goals.
+Thus, in order to address this use case adequately, the specifications
+for this working group must consider user experience and performance
+to allow for easy, streamlined customer utility data access.
+
+#### Examples of users in this use case:
+
+* Platforms that asses carbon emissions for enterprises (
```
Comment (op-ravi) [inline docs/use-cases.md:54]: a minor typo (asses --> assess)

File: docs/use-cases.md [hunk 11]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+for calculating carbon emissions involve obtaining customer utility data,
+such as electric and natural gas usage, meter locations, rate plans, etc.
+
+#### Needs:
+
+* **Standardized Data Formats and Access Procedures:**
+This data often needs to be gathered both as historical data
+(e.g. getting the past year of usage data for a historical analysis)
+and ongoing data (e.g. getting the past day's usage for real-time tracking
+of emissions). To address this need, the specifications from this working
```
Comment (op-ravi) [inline docs/use-cases.md:28]: The data standards should also specify the best practices for data storage and collection granularity or frequency. For example, the real-time data should have at least <5 minutes and historical data can be aggregated at hourly intervals.

File: docs/use-cases.md (hunk 1 above)
Comment (op-ravi) [reply docs/use-cases.md:46]: yes, I just added a similar comment above to include data granularity/frequency recommendations. Having a standard data frequency will be helpful for accurate and consistent data tracking and reporting.

File: docs/use-cases.md [hunk 12]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+Many organizations do not have dedicated staff or sufficient resources
+to spend large amounts of time and effort gathering utility data needed
+for carbon accounting calculations. This effectively creates a
+"barrier to entry" that limits carbon accounting to only the most well-resourced
+organizations and prevents widespread adoption of decarbonization goals.
+Thus, in order to address this use case adequately, the specifications
+for this working group must consider user experience and performance
+to allow for easy, streamlined customer utility data access.
+
+#### Examples of users in this use case:
```
Comment (op-ravi) [inline docs/use-cases.md:52]: Do we want to include the energy utilities as another user to collect and report data in a standardized way?

File: docs/use-cases.md [hunk 13]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+
+
+## Use Case 2: Decarbonization Projects <a id="use-case-decarbonization-projects" href="#use-case-decarbonization-projects" class="permalink">🔗</a>
+
+Often, organizations with decarbonization targets (see [Use Case 1: Carbon Accounting](#use-case-carbon-accounting) above),
+will evaluate projects that may positively impact their carbon emissions. These projects
+can range from physical locally (e.g. installing solar or EV charging) and remote (e.g.
+building renewable generation projects via wholesale power purchase agreements), to virtual
+direct (e.g. avoiding marginal emissions by managing building usage dynamically) or financial
+(e.g. switching rate plans to carbon free sources or community solar).
```
Comment (op-ravi) [inline docs/use-cases.md:79]: DAC (direct air capture)/ Carbon Capture projects are going to play an important role in the decarbonization journey, so I think we should include those into our scope.

File: docs/use-cases.md [hunk 14]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+
+* **Program Eligibility and Participation Information:**
+Because distributed grid flexibility and load management technologies are active
+participants in managing load on the electrical grid, they often are registered
+as part of a utility or central operator program (e.g. demand response programs).
+Thus, the specifications to address this use case must consider how to include
+program eligibility and participation details as part of the standardized customer
+dataset.
+
+#### Examples of users in this use case:
```
Comment (op-ravi) [inline docs/use-cases.md:148]: We should include DER Aggregator also to this list. DER Aggregators are going to be one of the important user in the decentralized and distributed grid.

File: docs/use-cases.md [hunk 15]
```diff
@@ -7,3 +7,216 @@ meta_description: What are the use cases that the Customer Data specifications t
...
+increasingly need sources of dynamic load flexibility to partially address
+the intermittency of solar and wind generation. Technologies to allow for
+higher and higher penetration of renewables on the grid can often be installed
+"behind the meter" as distributed energy resources (DERs).
+
+While these technologies do not directly decarbonize the grid, they allow for
+higher penetration of cheap, zero carbon renewables. Thus, this working group
+considers grid flexibility and load management technologies a key use case
+for bolstering decarbonization efforts.
+
```
Comment (op-ravi) [inline docs/use-cases.md:129]: We should include FERC order 2222 to support this use case. I understand the order.2222 is specific to the USA but significant enough to refer.
